### PR TITLE
[RBFDD-322] Extend Flink interpreter to work with YARN

### PR DESCRIPTION
**Pull request by alkagin**

### CODE REVIEW

**Comment by maocorte** [comment]
@alkagin good work!!
Do you think could be interesting also add the info of the 2 new environment variables also into [zeppelin env template](https://github.com/radicalbit/incubator-zeppelin/blob/rbp-1.0-zeppelin-0.6.0/conf/zeppelin-env.sh.template)?

File: flink/src/main/java/org/apache/zeppelin/flink/FlinkInterpreter.java [hunk 1]
```diff
@@ -224,7 +312,8 @@ public Object getValue(String name) {
   public void close() {
     flinkIloop.closeInterpreter();
 
-    if (localMode()) {
+    final String host = Strings.nullToEmpty(getProperty(HOST).trim());
```
Comment (alkagin) [inline flink/src/main/java/org/apache/zeppelin/flink/FlinkInterpreter.java]: use getHost()

File: flink/src/main/java/org/apache/zeppelin/flink/FlinkInterpreter.java [hunk 2]
```diff
@@ -69,41 +69,150 @@ public FlinkInterpreter(Properties property) {
...
+        inputStream = new FileInputStream(propertiesFile);
+        properties.load(inputStream);
+      } catch (IOException e) {
+        e.printStackTrace();
+      } finally {
+        if (inputStream != null) {
+          try {
+            inputStream.close();
+          } catch (IOException e) {
+            e.printStackTrace();
```
Comment (robbenti) [inline flink/src/main/java/org/apache/zeppelin/flink/FlinkInterpreter.java]: Avoid the usage of printStackTrace in production code, use the logger instead

**Comment by maocorte** [comment]
LGTM
Thank you for the contribution @alkagin

**Comment by robbenti** [comment]
LGTM
Much appreciated

**Pull request merged**